robosuite/renderers/igibson/igibson_utils.py

```diff
@@ -80,8 +80,10 @@ def load_object(
 
     if geom_type == "mesh":
         scale = geom_scale
-    elif geom_type in ["box", "sphere"]:
+    elif geom_type == "box": 
```

Comment:
Do these changes mean that we can create ellipsoids but we can only create cubes with same height/width/depth? Shouldn't we make the same change for the "box" case as it is done for sphere, i.e.,
`scale = np.repeat(geom_size * 2, 3)`
?

Reply:
I think we can create cubes with differing height/width/depth, since `BoxObject` is constructed with 3-dimensional `size`, the `geom_scale` value here will also be length 3 for the "box". But since `BallObject` has a 1-dimensional `size`, we need to repeat it here since iGibson expects a 3-dimensional `scale`. Currently we can't create ellipsoids (only spheres), but perhaps we could add an additional primitive class for ellipsoids with 3-dimensional `size`, or generalize the current `BallObject`.

Reply:
got it
sounds good!